app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,24 @@
  *= require font-awesome
  */
 
+
+.searchLocation {
+  height: 34px;
```

Comment:
Properties should be ordered background-color, background-image, border, border-radius, -webkit-box-shadow, box-shadow, color, font-size, height, line-height, padding, -o-transition, -webkit-transition, transition